lib/open_project/authentication.rb

```diff
@@ -0,0 +1,47 @@
+require 'open_project/authentication/manager'
+
+module OpenProject
+  ##
+  # OpenProject uses Warden strategies for request authentication.
+  module Authentication
+    class << self
+      ##
+      # Updates the used warden strategies for a given scope. The strategies will be tried
+      # in the order they are set here. Plugins can call this to add or remove strategies.
+      # For available scopes please refer to `OpenProject::Authentication::Scope`.
+      #
+      # @param [Symbol] scope The scope for which to update the used warden strategies.
+      # @param [Boolean] store Indicates whether the user should be stored in the session
+      #                        for this scope.
```

Comment:
The `store_defaults` value for that scope will remain unchanged which means what ever default value is [set by the Manager](https://github.com/machisuji/openproject/blob/feature/basic_auth/lib/open_project/authentication/manager.rb#L28) (`false`) will be used.